src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -342,7 +342,8 @@ public async Task Put()
                         [nameof(DatabasePutResult.RaftCommandIndex)] = newIndex,
                         [nameof(DatabasePutResult.Name)] = databaseRecord.DatabaseName,
                         [nameof(DatabasePutResult.Topology)] = topology.ToJson(),
-                        [nameof(DatabasePutResult.NodesAddedTo)] = nodeUrlsAddedTo
+                        [nameof(DatabasePutResult.NodesAddedTo)] = nodeUrlsAddedTo,
+                        [nameof(DatabasePutResult.ShardsDefined)] = databaseRecord.Shards != null && databaseRecord.Shards.Length >= 2
```

Comment:
I don't think it would be a problem if we have a single shard, I can think of reason why we might get there (manually resetting things). Let's not place this limit.
```suggestion
                        [nameof(DatabasePutResult.ShardsDefined)] = databaseRecord.Shards != null && databaseRecord.Shards.Length > 0
```

Reply:
done